### PR TITLE
refactor: join buttons components style into one

### LINKED ISSUE #50: UI Component | Fix buttons styling
Fix styles for component of ButtonRed & ButtonWhite..., because the css overrites and displays incorrect styling

### PR DESCRIPTION
**Pull request by vieraboschkova**
closes #50

### CODE REVIEW

File: styles/components/_buttons.scss [hunk 1]
```diff
@@ -35,6 +44,14 @@
     left: 8%; //check!
   }
 
+  &--red {
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25), 0px 4px 30px #ff0000;
+    background-color: #ff0000;
+    color: #fff;
```
Comment (DominikNowak) [inline styles/components/_buttons.scss:50]: Here can be also used variable from _variables.scss

File: styles/components/_buttons.scss [hunk 2]
```diff
@@ -1,30 +1,39 @@
...
-  position: relative;
   min-width: 400px;
+
+  // Color
+  background-color: $color-primary;
+  color: $text-color-primary;
+
+  // Text
+  font-weight: 500;
+  font-size: 2.5rem;
```
Comment (DominikNowak) [inline styles/components/_buttons.scss:19]: Same here

**Review by DominikNowak — APPROVED** [review]
It's ok for me, you may consider only using variables from _variables.scss unless you have reason not to. And lines 28 and 34 seems to be not formatted :P `npm run format`

File: styles/components/_buttons.scss [hunk 3]
```diff
@@ -1,30 +1,39 @@
 .button {
-  font-family: $font-primary;
+  // font-family: $font-primary;
```
[the remark this answers is not in the document]
Comment (vieraboschkova) [reply styles/components/_buttons.scss:2]: to be removed

**Review by SebastianBabinski1 — APPROVED** [review]
After changes it looks fine to me

**Pull request merged**